### ISSUE TITLE
Windows 11 Context menu change

### ISSUE THREAD

**Issue by Russkil** [ip7z/7zip#4]
can you update the software so on windows  11 context menu it shows before the show more options

https://www.reddit.com/r/Windows11/comments/12alrgw/move_7zip_from_show_more_options_to_appear_in/ - chat here

**Comment by inkydragon** [comment]
Just search "Windows 11 Classic Context Menu", there are many ways to go back to the old style.

**Comment by proudcanadianeh** [comment]
Would also love to see an update to add this natively to the new Windows 11 context menu. 
Expecting everyone to revert their windows to the classic menu is not a realistic option.

**Comment by bearonion** [comment]
You can use [NanaZIP](https://apps.microsoft.com/detail/9n8g7tscl18r) which integrates support for Windows 11 new context menu or this [workaround](https://gist.github.com/AveYo/305ec63e18518b22f41d1b52a3d5022b).

**Comment by Abdulaziz-IT** [comment]
Will this ever be added? We really need it.

**Comment by recolic** [comment]
> Just search "Windows 11 Classic Context Menu", there are many ways to go back to the old style.

Yes it's a great workaround, but eventually, at some time point, you have to add support to win11 context menu, sooner or later...

It's unavoidable and some user will find it useful even if we don't like it.

**Comment by amddsc** [comment]
> > Just search "Windows 11 Classic Context Menu", there are many ways to go back to the old style.
> 
> Yes it's a great workaround, but eventually, at some time point, you have to add support to win11 context menu, sooner or later...
> 
> It's unavoidable and some user will find it useful even if we don't like it.

Yes I agree, Winrar already has it.

**Comment by livxbox5** [comment]
When will Windows 11 context menu support be released?
Когда появится поддержка 
контекстное меню Windows 11?

**Comment by livxbox5** [comment]
When will Windows 11 context menu support be released?
Когда появится поддержка
контекстное меню Windows 11?

**Comment by 7OH** [comment]
26 версия и опять без поддержки вин 11.

**Comment by livxbox5** [comment]
Когда будет поддержка контекстного меню windows 11 ? @ip7z 
When will Windows 11 context menu support arrive? @ip7z

**Comment by Jakub-Cepela** [comment]
Adding my vote. Was forced to upgrade to Win 11 few months ago and still cannot get used to having 7zip in "hidden" menu.
@ip7z  Please consider adding proper support for Win11 context menus.

**Comment by recolic** [comment]
> Adding my vote. Was forced to upgrade to Win 11 few months ago and still cannot get used to having 7zip in "hidden" menu. [@ip7z](https://github.com/ip7z) Please consider adding proper support for Win11 context menus.

maybe we can try vibe coding and get a dirty (but working) PR by ourselves :)

**Comment by Grim1313** [comment]
+1 to the request to add 7Zip to the Windows 11 context menu.

**Comment by xandfis** [comment]
@ip7z if I submit a PR to add this support would you consider it, or are you opposed to adding this support?
